Meta/new-project.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+script_path=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+cd "${script_path}/.." || exit 1
+
+script_name=$(basename "$0")
+
+function list_templates() {
+  echo "Available templates:"
+  for file in ./Base/res/devel/templates/*.ini; do
+    printf '  %s - ' "$(basename "${file%%.ini}")"
+    awk -F "=" '/Description/ { print $2 }' "$file"
+  done
+}
+
+function usage() {
+  local return_code=${1:-0}
+
+  cat <<EOF
+Usage: $script_name TEMPLATE DESTINATION
+
+Instantiates a HackStudio template into a project at the given destination.
+The last component of the destination path is used as the project name, and must
+be a valid identifier (but hyphens are allowed and will be converted into
+underscores).
+
+Parameters:
+  TEMPLATE - The HackStudio template to use.
+  DESTINATION - The destination directory.
+
+$(list_templates)
+EOF
+  exit "$return_code"
+}
+
+while [[ $# -ge 1 ]]; do
+  case "$1" in
+    -h|--help) usage;;
+    -*) echo "$script_name: unknown parameter $1"; usage 1;;
+    *) break;;
+  esac
+done
+
+[[ $# -ne 2 ]] && echo "$script_name: takes exactly 2 parameters" && usage 1
+
+TEMPLATE="$1"
+DESTINATION="$2"
+
+TEMPLATE_SOURCE_DIRECTORY="./Base/res/devel/templates/$TEMPLATE"
+TEMPLATE_INI="./Base/res/devel/templates/$TEMPLATE.ini"
+TEMPLATE_POSTCREATE="./Base/res/devel/templates/$TEMPLATE.postcreate"
+
+if [[ ! -f "$TEMPLATE_INI" ]]; then
+  echo "$script_name: unknown template \"$TEMPLATE\"."
+  list_templates
+  exit 1
+fi
+
+PROJECT_NAME="$(basename "$DESTINATION")"
```

Comment:
Either quote these, or don't.
Let's pick one and stay with it (other instance at L8)

Reply:
Sticking with quoting. The others were leftovers from the script I copied them from. :^)